airbyte-integrations/connectors/source-adjust/source_adjust/source.py

```diff
@@ -0,0 +1,211 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import datetime
+import decimal
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import requests
+import source_adjust.model
+from airbyte_cdk.models import AirbyteMessage, AirbyteStateMessage, SyncMode, Type
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import IncrementalMixin, Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.requests_native_auth import TokenAuthenticator
+
+
+class AdjustStream(HttpStream, IncrementalMixin):
```

Comment:
I noticed that Adjust also has API endpoints for ad spent and KPI. Should we change the stream name to `AdjustReportStream` to be more explicit??